scss/mixins/_utilities.scss

```
@@ -2,81 +2,71 @@
 // Used to generate utilities & print utilities
 @mixin generate-utility($utility, $infix: "", $is-rfs-media-query: false) {
   $values: map-get($utility, values);
+  $bootstrap-compatibility: map-get($utility, bootstrap-compatibility); // OUDS mod
 
-  // If the values are a list or string, convert it into a map
-  @if type-of($values) == "string" or type-of(nth($values, 1)) != "list" {
-    $values: zip($values, $values);
-  }
-
-  @each $key, $value in $values {
-    $properties: map-get($utility, property);
-
-    // Multiple properties are possible, for example with vertical or horizontal margins or paddings
-    @if type-of($properties) == "string" {
-      $properties: append((), $properties);
+  @if ($bootstrap-compatibility != true or $enable-bootstrap-compatibility == true) { // OUDS mod: filter Bootstrap specificities
```

It's pretty obvious so we can probably drop the detail of the comment:
```suggestion
  @if ($bootstrap-compatibility != true or $enable-bootstrap-compatibility == true) { // OUDS mod
```